build/build.ps1

```diff
@@ -181,9 +184,49 @@ if (!(Test-Path $CAKE_EXE)) {
     Throw "Could not find Cake.exe at $CAKE_EXE"
 }
 
+# Make sure modules folder exists
```

Comment:
Why these changes?

Reply:
@nmetulev See dcd5aff451fdbd1d7e7e77d785c55071145c4f23. Later replaced with 174ee69cc1abb8ed11658c18580464bb7c070745 but maintained change. Can revert this portion since I'm using the directive introduced with the latter.